codechain/codechain.yml

```diff
@@ -46,6 +46,11 @@ args:
         long: chain
         help: Set the blockchain type out of solo, solo_authority, tendermint, cuckoo, blake_pow, husky or a path to chain scheme file.
         takes_value: true
+    - wait-before-shutdown:
```

Comment:
I don't think we need a command line option here because this PR is just a hotfix for the crash. Just use a hardcoded constant which is large enough to avoid the crash.

Reply:
It's to help test.
I'm not sure waiting 1 second is enough to avoid the crash. It was the experimental result on my machine.

Reply:
I think we need internal command line options to help test and debug.

Reply:
It is not a good idea to expose a command line option that is used only for test & debug purposes. What about adding an environment variable instead? Users don't know what environment variables exist, but we can use them for test & debug purposes.

Reply:
Okay. I agree with you. I'll change it.